examples/city_walking_behaviour/city_walking_behaviour/model.py

```diff
@@ -0,0 +1,729 @@
+import math
+
+from mesa import Model
+from mesa.datacollection import DataCollector
+from mesa.experimental.cell_space import OrthogonalVonNeumannGrid
+from mesa.experimental.cell_space.property_layer import PropertyLayer
+from mesa.experimental.devs import ABMSimulator
+
+from .agents import (
+    DOG_OWNER_PROBABILITY,
+    FEMALE_PROBABILITY,
+    MAX_AGE,
+    MIN_AGE,
+    SINGLE_HOUSEHOLD_PROBABILITY,
+    GroceryStore,
+    Human,
+    NonFoodShop,
+    Other,
+    SocialPlace,
+)
+
+SCENARIOS = {
+    "random_random": "Random Land Use, Random Safety",
+    "random_safe": "Random Land Use, Low Safety in Core",
+    "central_random": "Centralized Land Use, Random Safety",
+    "central_safe": "Centralized Land Use, Low Safety in Core",
+}
+
+
+class WalkingModel(Model):
+    def __init__(
+        self,
+        height: int = 40,
+        width: int = 40,
+        no_of_couples: int = 2400,
+        no_of_singles: int = 600,
+        no_of_grocery_stores: int = 10,
+        no_of_social_places: int = 75,
+        no_of_non_food_shops: int = 40,
+        no_of_others: int = 475,
+        scenario: str = "random_random",
+        seed=None,
+        simulator=ABMSimulator(),
```

Comment:
Are you actually using DEVS functionality? I can't find it in your code, so I'm curious if you use the functionality.